.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -4,7 +4,19 @@ about: Report a bug encountered with the gateway-api project
 labels: kind/bug
 
 ---
-<!-- Please use this template while reporting a bug and provide as much info as
+<!--
+Note: Bug reports here should only be used for bugs with the API itself, such
+as:
+
+- Incomplete or inaccurate validation
+- Problems with the API specification
+- Unexpected errors with the validating webhook
+- Flaws in conformance tests
+
+Bugs that don't fit into one of the above categories should likely be filed
+with the implementation of Gateway API you're using.
```

Comment:
```suggestion
with the [implementation](https://gateway-api.sigs.k8s.io/implementations/) of Gateway API you're using.
```

The implementations page might help the reporter find where to file their issue

Reply:
Updated, thanks!